vignettes/savvyr_example.Rmd

```diff
@@ -0,0 +1,114 @@
+---
+title: "Estimation of AE probabilities with savvyr"
+package: savvyr
+bibliography: "../inst/REFERENCES.bib"
+output:
+  rmarkdown::html_vignette:
+          toc: true
+vignette: |
+  %\VignetteEncoding{UTF-8}
+  %\VignetteIndexEntry{Estimation of AE probabilities with savvyr}
+  %\VignetteEngine{knitr::rmarkdown}
+editor_options:
+  chunk_output_type: console
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+library(savvyr)
+library(kableExtra)
+```
+
+
+
+
+# Example using dummy data
+
+We generate the dataset $S1$ in  @stegherr_meta_analytic_2021 using the parameter values for Arm A:
+
+```{r, include=TRUE, echo=TRUE}
+# sample size
+N <- 200
```

Comment:
maybe here try to comply with linter, so e.g. here
```suggestion
n <- 200
```

Reply:
adapted